Formula/fontforge.rb

```diff
@@ -16,71 +16,72 @@ class Fontforge < Formula
 
   deprecated_option "with-gif" => "with-giflib"
 
-  # Autotools are required to build from source in all releases.
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
+  if build.head?
```

Comment:
Make a `head do` block and put these in there along with the above `head` as a `url` instead.